microprofile/jwt-auth/src/main/java/io/helidon/microprofile/jwt/auth/JwtAuthAnnotationAnalyzer.java

```diff
@@ -102,30 +102,18 @@ public AnalyzerResponse analyze(Class<?> maybeAnnotated) {
     // resource class analysis
     @Override
     public AnalyzerResponse analyze(Class<?> maybeAnnotated, AnalyzerResponse previousResponse) {
-        return performAnalysis(maybeAnnotated, previousResponse);
+        return performAnalysis(previousResponse);
```

Comment:
Please remove the method `performAnalysis` as it is now useless and confusing.

Reply:
Done